### PR TITLE
API resources improvement

### PR DESCRIPTION
**Pull request by sherubthakur**

### CODE REVIEW

File: kanapy/resources.py [hunk 1]
```diff
@@ -7,6 +7,11 @@
 DICT = type({})
 
 
+def to_upper_camel(snake):
+    lst = snake.split("_")
+    return "".join(map(lambda x: x[0].upper() + x[1:], lst))
```
Comment (dhruvagarwal) [inline kanapy/resources.py]: use Capitalize instead

File: kanapy/resources.py [hunk 2]
```diff
@@ -57,20 +54,14 @@ def delete(cls, id_):
         raise NotImplementedError()
 
     @classmethod
-    def get(cls, id):
+    def get(cls, c, id=None):
```
Comment (dhruvagarwal) [inline kanapy/resources.py]: accepting it and defining it in the next line?

File: kanapy/resources.py (hunk 1 above)
Comment (kashifrazzaqui) [reply kanapy/resources.py]: No, that won't work - its a different usecase

Comment (dhruvagarwal) [reply kanapy/resources.py]: I meant `x: x.capitalize()`

Comment (sherubthakur) [reply kanapy/resources.py]: capitalize will do `classA` to `Classa` but I want it to be `ClassA`

**Pull request merged**